### PR TITLE
OpenPaaS-Suite/esn-frontend-admin#10 added MAX_SIZE_UPLOAD_DEFAULT const

### PR DESCRIPTION
**Pull request by rezk2ll**
closes https://github.com/OpenPaaS-Suite/esn-frontend-admin/issues/10

### CODE REVIEW

**Review by asmaNode — DISMISSED** [review]

File: src/frontend/js/modules/file.js [hunk 1]
```diff
@@ -5,6 +5,7 @@
 
   .constant('FILES_API_URL', '/api/files')
   .constant('DEFAULT_FILE_TYPE', 'application/octet-stream')
+  .constant('MAX_SIZE_UPLOAD_DEFAULT', 104857600)
```
Comment (MichaelBailly) [inline src/frontend/js/modules/file.js]: we lost the comment (size in Mb)

**Review by piavgh — DISMISSED** [review]

**Review by MichaelBailly — CHANGES_REQUESTED** [review]

File: src/frontend/js/modules/file.js (hunk 1 above)
Comment (rezk2ll) [reply src/frontend/js/modules/file.js]: now it's back

**Review by MichaelBailly — APPROVED** [review]

File: src/frontend/js/modules/file.js [hunk 2]
```diff
@@ -5,6 +5,7 @@
 
   .constant('FILES_API_URL', '/api/files')
   .constant('DEFAULT_FILE_TYPE', 'application/octet-stream')
+    .constant('MAX_SIZE_UPLOAD_DEFAULT', 104857600) // is equal to 100 MO
```
Comment (piavgh) [inline src/frontend/js/modules/file.js:8]: What is MO?

Comment (rezk2ll) [reply src/frontend/js/modules/file.js:8]: Mega Octet, works both for English and French.

**Review by piavgh — APPROVED** [review]

**Pull request merged**